File: hi.go
package main

import "fmt"

func main() {
	ch := make(chan int,1000)
	go sender(ch)
	for i := 0; i < 100000; i++ {
		ch <- i
	}
	close(ch)
}

func sender(ch chan int) {
	for pressure := range ch {
		fmt.Println(pressure)
	}
	return
}

File: hi.c
/* Hello World program */

#include<stdio.h>

main() {
	//ch := make(chan int,1000)
	//go sender(ch)
	int i = 0;
	for (; i < 100000; i++) {
		//ch <- i
		printf("%d\n",i);
	}
	//close(ch)
}